frame/dvm/evm/precompiles/utils/Cargo.toml

```diff
@@ -11,16 +11,22 @@ version     = "2.8.11"
 
 [dependencies]
 # crates.io
-ethabi = { version = "15.0", default-features = false }
-evm    = { version = "0.33.1", default-features = false, features = ["with-codec"] }
+ethabi       = { version = "15.0", default-features = false }
+ethereum     = { version = "0.11.1", features = ["with-codec"], optional = true }
+evm          = { version = "0.33.1", default-features = false, features = ["with-codec"] }
+libsecp256k1 = { version = "0.5", features = ["static-context", "hmac"], default-features = false, optional = true}
+rlp          = { version = "0.5", optional = true}
+sha3         = { version = "0.9", default-features = false, optional = true}
```

Comment:
should some of the above packages be moved to the dev-dependencies?

Reply:
I checked it again, some deps are under `testing` feature control.